package.json

```diff
@@ -15,42 +15,46 @@
     "@sentry/react": "^6.18.2",
     "@sentry/tracing": "^6.18.2",
     "@sushiswap/sdk": "5.0.0-canary.116",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
+    "@types/ethereum-block-by-date": "^1.4.1",
```

Comment:
`dependencies` now contains everything required to build the app - and nothing more. This should speed up the Fleek build time a bit.

Reply:
LFG!!!!

Reply:
same with eslint and what not

Reply:
@wraeth-eth  yesss I had the same initial thought, but because the build script does typescript compilation and also eslint checks, we need em all in `dependencies`.

So pretty much everything except **dev-only** deps (like source map analyzer and whatnot) is a `dependency` of the app atm.

Reply:
ok cool cool TIL sounds good